### PR TITLE
Commiting

### PR DESCRIPTION
**Pull request by ShivaPandey02**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,32 @@
...
   const [news, setNews] = useState([]);
-  var apiKey = `db01e884d61f4f908c66f1200c039fc4                                                                                                `;
-  const [searchQuery, setSearchQuery] = useState(["Trending"]);
+  var apiKey = `db01e884d61f4f908c66f1200c039fc4`;
+  const [searchQuery, setSearchQuery] = useState(["Trending World Wide"]);
   const [loading, setLoading] = useState(false);
-  //const [url, setUrl] = useState([`http://api.mediastack.com/v1/news?access_key=c35fdbfb0c5a2e16c400bbd2dbd3bca5&keywords=spiderman&sort=published_desc&limit=50&languages=en`])
   const [url, setUrl] = useState([
-    `http://newsapi.org/v2/everything?q=Trending&language=en&sortBy=publishedAt&apiKey=${apiKey}`,
+    `http://newsapi.org/v2/everything?q=Trending World Wide&language=${lang}&sortBy=publishedAt&apiKey=${apiKey}`,
```
Comment (Shashank-Shastri) [inline src/App.js:12]: lang is not defined

Comment (ShivaPandey02) [reply src/App.js:12]: We can chat here??

Comment (ShivaPandey02) [reply src/App.js:12]: I had defined it
image

Comment (Shashank-Shastri) [reply src/App.js:12]: Sorry didn't see the lang definition above. But it's better if it's in App().

**Pull request merged**